### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by jiqing-feng**
Hi @amyeroberts 

We need to update some lib versions for CPU training in the docs, would you please help review it? Thx!

### CODE REVIEW

**Comment by sywangyi** [comment]
@stevhliu please help review the doc

**Review by amyeroberts — APPROVED** [review]
Thanks for updating! 

Changes look fine to me cc @SunMarc to confirm docs are in-line with recommended usage

**Review by SunMarc — APPROVED** [review]
LGTM ! This is in line with the recommended usage.

File: docs/source/en/perf_train_cpu.md [hunk 1]
```diff
@@ -31,14 +31,16 @@ IPEX release is following PyTorch, to install via pip:
...
 | 1.13              |  1.13.0+cpu    |
 | 1.12              |  1.12.300+cpu  |
-| 1.11              |  1.11.200+cpu  |
-| 1.10              |  1.10.100+cpu  |
 
+Please run `pip list | grep torch` to get your `pytorch_version`, so you can get the `IPEX version_name`.
 ```
 pip install intel_extension_for_pytorch==<version_name> -f https://developer.intel.com/ipex-whl-stable-cpu
 ```
+**___Note: Please check the upgraded versions in [ipex-whl-stable-cpu](https://developer.intel.com/ipex-whl-stable-cpu) if needed.___**
```
Comment (stevhliu) [inline docs/source/en/perf_train_cpu.md:43]:
```suggestion
You can check the latest versions in [ipex-whl-stable-cpu](https://developer.intel.com/ipex-whl-stable-cpu) if needed.
```

File: docs/source/en/perf_train_cpu.md [hunk 2]
```diff
@@ -59,8 +61,20 @@ Take an example of the use cases on [Transformers question-answering](https://gi
 --max_seq_length 384 \
 --doc_stride 128 \
 --output_dir /tmp/debug_squad/ \
-<b>--use_ipex \</b>
-<b>--bf16 --no_cuda</b></pre> 
+<b>--use_ipex</b> \
+<b>--bf16</b> \
+<b>--use_cpu</b></pre> 
+
+If you want to enable `use_ipex` and `bf16` in your script, just add some arguments in `TrainingArguments` like this:
```
Comment (stevhliu) [inline docs/source/en/perf_train_cpu.md:68]: Just to be more clear :)

```suggestion
If you want to enable `use_ipex` and `bf16` in your script, add these parameters to `TrainingArguments` like this:
```

**Review by stevhliu — APPROVED** [review]
Thanks for the update!

**Comment by jiqing-feng** [comment]
Hi @stevhliu . Thanks for your advice, I have fixed these comments. Please take a look, thx!

**Pull request merged**